### ISSUE TITLE
feature: add custom verbs/nouns etc

### ISSUE THREAD

**Issue by brettmc** [spencermountain/nlp_compromise#63]
Scenario: using the library for natural language processing for a calendar assistant. Doesn't recognise "schedule" as a verb.

Would it be possible to pass in some configuration when instantiating the library, eg an array of verbs + nouns etc to allow users to inject extra words.
In my case I might extend verbs by passing in an array of my own verbs:
`["schedule"]`

That *seems* to work for me if I hack the code and add 'schedule' to the list of verbs...but I don't grok grammar well enough to know if it's completely correct (it becomes an infinitive verb, VBP)

**Comment by spencermountain** [comment]
hey brett, this is a good idea. A few of us are working on a v.2, so it's also a good time to think about this very reasonable feature. I'll keep this open
cheers

**Comment by AloisReitbauer** [comment]
Hey,

I seem to have the same issue. I need "root cause" to be understood as a single noun. Right now it is detected as a noun and a verb. Where would be the best place to "hack" this in?

// Alois

**Comment by spencermountain** [comment]
hey alois, you should be able to add it here, and go
https://github.com/spencermountain/nlp_compromise/blob/master/src/data/lexicon/multiples.js#L74

**Comment by AloisReitbauer** [comment]
Seems the link is no longer valid.

**Comment by spencermountain** [comment]
yeah, am a lot closer to having this done, but isn't quite there. will have a per-method lexicon soon.
here's the link for that temporary hack now
https://github.com/nlp-compromise/nlp_compromise/blob/master/src/data/multiples.js
cheers

**Comment by kanishk-mehta** [comment]
Hi,
That link's broken too @spencermountain .. could you please guide me to the right link? .. Thank-you

**Comment by spencermountain** [comment]
hi @kanishk-mehta here's the documentation about adding new words
https://github.com/nlp-compromise/compromise/wiki/Lexicon
cheers

**Comment by kanishk-mehta** [comment]
Hey @spencermountain..Thanks! Sorry for the delayed response..
